python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -118,7 +112,7 @@ def initAlgorithm(self, config=None):
             QgsProcessingParameterNumber(
                 self.TILE_SIZE,
                 self.tr('Tile size'),
-                defaultValue=1024))
+                defaultValue=1024, minValue=0))
```

Comment:
Let's go with 64. I don't think there's a valid use case for 1 pixel wide tiles either ;)